bindata/oauth-apiserver/apiserver-clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
did you use the bindata from https://github.com/openshift/cluster-authentication-operator/pull/238? I think @sttts tested what he had there

Reply:
Ah, I see you did, good :+1: